bedrock/settings/__init__.py

```diff
@@ -259,31 +259,59 @@
         "connect-src": list(set(_csp_default_src + _csp_connect_src)),
         # support older browsers (mainly Safari)
         "frame-src": _csp_child_src,
-        "frame-ancestors": [csp.constants.NONE],
     },
 }
-
-# Start report-only CSP as a copy. We'll modify it later if needed.
 # Only set up report-only CSP if we have a report-uri set.
 if csp_report_uri := config("CSP_REPORT_URI", default="") or None:
     CONTENT_SECURITY_POLICY_REPORT_ONLY = deepcopy(CONTENT_SECURITY_POLICY)
+    # Add reporting.
     CONTENT_SECURITY_POLICY_REPORT_ONLY["REPORT_PERCENTAGE"] = config("CSP_REPORT_PERCENTAGE", default="100", parser=int)
     CONTENT_SECURITY_POLICY_REPORT_ONLY["DIRECTIVES"]["report-uri"] = csp_report_uri
+    # CSP directive updates we're testing that we hope to move to the enforced policy.
+    CONTENT_SECURITY_POLICY_REPORT_ONLY["DIRECTIVES"]["frame-ancestors"] = [csp.constants.NONE]
+    CONTENT_SECURITY_POLICY_REPORT_ONLY["DIRECTIVES"]["style-src"].remove(csp.constants.UNSAFE_INLINE)
```

Comment:
FYI the PR description states "Remove <mark>`script-src`</mark> `unsafe-inline` from report-only for testing", not `style-src` — not that it would be _too_ important, only for someone keeping track of what is taken care of and what still needs sorting out this might throw them off a bit;)

Reply:
Oops, good catch. I can update the PR comment but not the git commit.

Reply:
I think we're getting there slowly;) Thanks for updating it — so it doesn't add to the plethora of CSP moving targets;)) 🚀